packages/form-js-editor/src/features/properties-panel/entries/factories/simpleBoolEntryFactory.js

```diff
@@ -1,5 +1,5 @@
 import { get } from 'min-dash';
-import { CheckboxEntry, isCheckboxEntryEdited } from '@bpmn-io/properties-panel';
+import { ToggleSwitchEntry, isToggleSwitchEntryEdited } from '@bpmn-io/properties-panel';
```

Comment:
I've already made this change somewhere else, maybe dynamic list, anyways +1

Reply:
Let's say I got inspired 😅